app/ui/browser/actions/main-actions.js

```diff
@@ -137,17 +104,12 @@ export function setUserTypedLocation(pageId: string, payload: Object): Action {
     // Empty input could happen if a page finishes loading and the userTyped
     // state is going to be reset.
     if (payload.text) {
-      userAgent.api(`/query?q=${encodeURIComponent(payload.text)}`, {
-        method: 'GET',
-      })
-        .then(async function (response) {
-          if (!response.ok) {
-            return;
-          }
-          const body = await response.json();
-          dispatch(profileDiffs.completions(payload.text, body.results));
-        })
-        .catch(); // Right now, nothing to be done.  In the future, we could retry.
+      userAgent.query({ text: payload.text }).then(res => {
```

Comment:
Initial thought was that's extra processing if we don't want it, but that is trivial -- will add that